schemas/credentials.json

```diff
@@ -365,6 +365,7 @@
                 "title": "SMTP username",
                 "description": "Username for SMTP authentication",
                 "default": "",
+                "encrypt": true,
```

Comment:
Other examples in this schema have usernames as `"encrypt": false,`. Usernames should follow the established pattern.